packages/http-client-python/package.json

```diff
@@ -41,7 +41,8 @@
     "ci": "npm run test:emitter && npm run ci:generator --",
     "ci:generator": "tsx ./eng/scripts/ci/run-ci.ts",
     "test:generator": "tsx ./eng/scripts/ci/run-ci.ts --command=test",
-    "test:emitter": "vitest run -c ./emitter/vitest.config.ts"
+    "test:emitter": "vitest run -c ./emitter/vitest.config.ts",
+    "change": "chronus"
```

Comment:
and yeah for merging it means we definitely need to find a solution for releasing our packages without yours.

Can you actually add a new policy in chronus.yaml that is either `isolated` with type `independent` or call it `python-client`) so I can add some exclude policy flag and we can update our release script

Reply:
isn't the pr also missing the removal from chronus.yaml

Reply:
can you lmk if the change i just added makes sense in the chronus yaml? thanks!